extensions/README.md

```diff
@@ -260,20 +260,24 @@ See [Publicize](blocks/publicize/index.js) and [Shortlinks](blocks/shortlinks/in
 
 ### i18n
 
-As of 04/2019, `wp.i18n` [doesn't support React elements in strings](https://github.com/WordPress/gutenberg/issues/9846). You will have to structure your copy so that links and other HTML can be translated separately.
-
-Not possible:
-
-```js
-__( 'Still confused? Check out <a>documentation</a> for more!' )
-```
-
-Possible:
+`@wordpress/i18n` doesn't support React elements in strings, but you can use `__experimentalCreateInterpolateElement` from `@wordpress/element`:
 
 ```jsx
-{ __( 'Still confused?' ) } <a>{ __( 'Check out documentation for more!' ) }</a>
+import { BlockIcon } from '@wordpress/block-editor';
+import { __experimentalCreateInterpolateElement as createInterpolateElement } from '@wordpress/element';
```

Comment:
I'd assume this, but does `__experimentalCreateInterpolateElement` work in newer versions of Gutenberg plugin as well? There's no need to check if `createInterpolateElement` exists?

Reply:
I think `__experimentalCreateInterpolateElement ` actually no longer exists since https://github.com/WordPress/gutenberg/pull/20699

```suggestion
import { createInterpolateElement } from '@wordpress/element';
```

Reply:
Oh, I thought those `__experimental` APIs stayed around for a bit longer. That's problematic. We can't just import `createInterpolateElement ` since Jetpack has to remain compatible with WordPress 5.4, that did not include it yet:
https://github.com/WordPress/gutenberg/blob/wp/5.4/packages/element/src/index.js

Reply:
I updated this PR in 80c5792f156f0ef43e2edc7f2785a1e2de48b38f based on the above, and based off the discussion here:
p1594723325477500-slack-gutenberg

Reply:
@yuliyan suggested we could do something like this:
```
import { createInterpolateElement, __experimentalCreateInterpolateElement } from '@wordpress/element';
const createInterpolateElementFn = createInterpolateElement || __experimentalCreateInterpolateElement;
```
We could write a very small wrapper to hide this workaround while it's still needed?

Reply:
That's a good idea. I updated this PR with fa27936b48774ea9eba91d561ca19dec83975eaa.